src/connectors/answers/__tests__/connectAnswers-test.ts

```diff
@@ -0,0 +1,447 @@
+import algoliasearchHelper, { SearchResults } from 'algoliasearch-helper';
+import { createInstantSearch } from '../../../../test/mock/createInstantSearch';
+import { createSearchClient } from '../../../../test/mock/createSearchClient';
+import {
+  createInitOptions,
+  createRenderOptions,
+} from '../../../../test/mock/createWidget';
+import { createSingleSearchResponse } from '../../../../test/mock/createAPIResponse';
+jest.mock('../../../lib/utils/debounce', () => ({
+  debounce: jest.fn(callback => (...args) => setTimeout(callback(...args), 0)),
+}));
+import { debounce } from '../../../lib/utils/debounce';
+import connectAnswers from '../connectAnswers';
+
+const wait = (time = 10) => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve();
+    }, time);
```

Comment:
```suggestion
    setTimeout(resolve, time);
```

This should avoid the `missing args` warning on `resolve()`